es/software.qmd

```diff
@@ -0,0 +1,77 @@
+---
+bibliography: ../software.bib
+csl: ../apa-7th-curriculum-vitae.csl
+nocite: |
+  @*
+description: |
+  Lista de paquetes de software y herramientas desarrolladas por el Doctor George G. Vega Yon
+---
+
+::: {style="text-align: center;"}
+![](../img/hexlogos.webp){width="70%" alt="Un conjunto de logotipos hexagonales del software en el que estoy trabajando"}
+:::
+
+# Software
+
+Estoy profundamente comprometido con el **software de código abierto** y he estado construyendo herramientas de nivel profesional durante más de 15 años—remontándose a los días de [Google Code aquí](https://code.google.com/archive/p/rdspl/){target="_blank"}. Mi trabajo combina métodos estadísticos rigurosos con prácticas modernas de ingeniería de software, incluyendo **control de versiones** (Git/GitHub), **integración continua**, **IA agéntica**, **tecnologías de contenedores** y **metodologías ágiles**. He liderado equipos de desarrollo de software científico para entregar herramientas **de alto rendimiento**, **confiables** y **fáciles de usar** que son ampliamente adoptadas por investigadores y profesionales. La mayor parte de mi trabajo está construido en C++ con wrappers para R y Python, optimizado para su uso en entornos de computación de alto rendimiento. En todos los proyectos, diseño con el rendimiento y la usabilidad en mente, asegurando que los métodos complejos puedan ser tanto escalables como accesibles.
+
+Un par de proyectos a destacar incluyen:
+
+
+```{r}
+#| label: cards
+#| results: asis
+#| echo: false
+#| eval: true
+x <- yaml::read_yaml("../software_highlights.yaml")
```

Comment:
I think that the `../software_highlights.yaml` should have individual versions for each language. Currently, the yaml file includes a path to a figure that is not displaying properly in the subsites:

<img width="844" height="666" alt="image" src="https://github.com/user-attachments/assets/f6e1c3df-5af7-4661-906a-293327a5190a" />

So, instead of `../software_highlights.yaml`, it should be `software_highlights.yaml` and update the file so the image reference is copied properly.